### ISSUE TITLE
Packagist not updated

### ISSUE THREAD

**Issue by asaliev** [josegonzalez/cakephp-queuesadilla#29]
@josegonzalez 
Could you try forcing packagist.org to update the package version? Looks like it's stuck on 0.5.1

**Comment by josegonzalez** [comment]
Yeah maybe the between us and packages broke, I'll add a todo for myself for later today. Apologies.

**Comment by makamo** [comment]
Is it solved?

**Comment by asaliev** [comment]
@makamo yeah, packagist shows 0.6 as latest version
